TabloidCLI/Repositories/PostRepository.cs

```diff
@@ -47,8 +47,49 @@ public List<Post> GetAll()
 
         public Post Get(int id)
         {
-            throw new NotImplementedException();
-        }
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"SELECT *
+                                          FROM Post 
+                                         WHERE Id = @id";
+
+                    cmd.Parameters.AddWithValue("@id", id);
+
+                    Post post = null;
+
+                    SqlDataReader reader = cmd.ExecuteReader();
+                    while (reader.Read())
+                    {
+                        if (post == null)
+                        {
+                            post = new Post()
+                            {
+                                Id = reader.GetInt32(reader.GetOrdinal("Id")),
+                                Title = reader.GetString(reader.GetOrdinal("Title")),
+                                Url = reader.GetString(reader.GetOrdinal("Url")),
+                                PublishDateTime = reader.GetDateTime(reader.GetOrdinal("PublishDateTime")),
+                            };
+                        }
+                        //this may be helpful later
+                        //if (!reader.IsDBNull(reader.GetOrdinal("TagId")))
```

Comment:
let's take out the zombie code. If you need to get it from the Author repo later to copy over, it's still there.

Reply:
I can remove it if you want. I left this in for when the next tickets are going to need the FKs.